Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM debian:bullseye as base
+
+# Set shell
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+ARG MATTER_SERVER_VERSION=1.0.7
+ARG HOME_ASSISTANT_CHIP_VERSION=2022.11.1
+
+WORKDIR /app
+
+RUN \
+    set -x \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+    libuv1 \
+    openssl \
+    zlib1g \
+    libjson-c5 \
+    python3-venv \
+    python3-pip \
+    python3-gi \
+    python3-gi-cairo \
+    python3-dbus \
+    python3-psutil \
+    unzip \
+    libcairo2 \
+    gdb \
+    git \
+    && git clone --depth 1 -b v1.0-branch \
```

Comment:
we grab the certificates from master branch

Reply:
```suggestion
    && git clone --depth 1 -b master \
```